src/components/Header.js

```diff
@@ -10,7 +10,6 @@ class Header extends React.Component {
       <Segment className="header_segment"
         inverted
         textAlign='center'
```

Comment:
Not sure it's a blocking change, but it might be worth checking if this kind of stuff could be abstracted too. Not sure if it does something complicated css-wise in semantic-ui-react.

Reply:
Good point, I was wondering the same thing myself. I'll have a look at the semantic-ui docs tomorrow and experiment, see if this can also be ported over the the stylesheet, Most of it seems pretty common sense.